File: java/program/Elektronik.java
public class Elektronik extends Produk {
    protected String merk;
    protected String daya;
    protected double harga;

    public Elektronik(String kodeProduk, String nama, int stok, String merk, String daya, double harga) {
        super(kodeProduk, nama, stok);
        this.merk = merk;
        this.daya = daya;
        this.harga = harga;
    }
}

File: python/program/elektronik.py
from produk import Produk

class Elektronik(Produk):
    def __init__(self, kode_produk, nama, stok, merk, daya, harga):
        super().__init__(kode_produk, nama, stok)
        self.merk = merk
        self.daya = daya
        self.harga = harga

